stac_ipyleaflet/stac_discovery/stac.py

```diff
@@ -16,14 +18,20 @@ class OutputCollectionObj(TypedDict):
     description: str
     license: str
 
-class Stac():
 
+class Stac:
     @staticmethod
     def organize_collections(collections=[]):
         output_collections = []
         for collection in collections:
             try:
                 data = collection.to_dict()
+
+                has_cog = False
+                # determine if collection has compatible items by looking to its item_assets property
+                if data["item_assets"]:
+                    has_cog = True
```

Comment:
⛏️ : This possibly will define it twice, maybe a ternary?
`has_cog = True if data["item_assets"] else False`